### DISCUSSION TITLE
does librechat have inbuilt support for aws rabbit mq?

### DISCUSSION THREAD

**Discussion by iwgth** [danny-avila/LibreChat#4030] [Help Wanted]
Just want to hook up an aws message broker to use with librechat, and before I start, want to know if there are any inbuilt features for this. or how I can make it

**Comment by danny-avila** [comment]
No there are not

**Reply by iwgth** [reply]
perhaps you know how to connect the rabbit mq librechat correctly?
I'm using lightlm along with aws berdock - llama 3 model and I'm not sure I understand how to properly connect rabbit mq to all this.

**Reply by danny-avila** [reply]
What's the point to add rabbit mq? I suppose if you needed to add this, you would know where in terms of function.

Also, you don't need to use litellm anymore for bedrock: https://www.librechat.ai/docs/configuration/pre_configured_ai/bedrock

This is in case you care about inference speed as well as network transfer speed for tokens between your server and the users.
